### ISSUE TITLE
Problem with `makemessages` after updating to django-jinja 2.7.0

### ISSUE THREAD

**Issue by pacahon** [niwinz/django-jinja#272]
Hello. 

Looks like it's impossible to use `{% trans myvar %}` syntax in DTL files after this change https://github.com/niwinz/django-jinja/commit/e3764f65560c9e9edab71b858e345cc641621441 was introduced.

**Comment by auvipy** [comment]
@wizpig64 can you check please?

**Comment by hugokernel** [comment]
Same problem here.

Could you check please.

**Comment by wizpig64** [comment]
**TL;DR workaround for django 3.1 users**:

[Django 3.1 renamed the `trans` tag family to `translate`, but kept the old tags around for compatibility with old templates.](https://docs.djangoproject.com/en/3.1/releases/3.1/#templates)

If you are using django 3.1 or higher, convert your DLT templates to use `{% translate myvar %}` instead of `{% trans myvar %}`, and things ought to work again using the current django-jinja 2.7.0.  

---

**the part you don't have to read, but you can if you want to solve this regex puzzle**:

I was able to replicate this issue by converting the repo's `i18n_test.jinja` to DTL. At first it looked like nothing was wrong, but after re-reading the ticket i realized i was using `{% translate %}` and not `{% trans %}`. switching got it to raise the initially-misleading message `SyntaxError: Translation blocks must not include other block tags: trans 'Year' as t_year (file testapp/templates/i18n_test.html, line 6)`.

Here's what's going on: Django is trying to prevent nested `{% blocktrans(late) %}{% endblocktrans(late) %}` blocks, but the name of jinja's equivalent to these (`{% trans %}`/`{% endtrans %}`) collides with django's standalone `{% trans %}` tag. Because our makemessages is simply [monkey patching](https://github.com/niwinz/django-jinja/blob/master/django_jinja/management/commands/makemessages.py#L1-L25) django's makemessages, it processes both types of templates with the same set of regexes, and thus it mistakenly treats a DTL `{% trans %}` tag like a `{% blocktrans(late) %}` tag, then failing because it expects a matching `{% endblocktrans(late) %}`.

Since this occurred because of a regex change, i think this is fixable, but i don't want to just revert the change and break #255 again.

Will keep poking at this, but it turns out regex is hard!

**Comment by wizpig64** [comment]
@pacahon @hugokernel I've submitted a change that fixed this in my test template, and should not necessitate upgrading to the `translate` tag as above. Please test it out with your templates, as they might break differently than mine did.

**Comment by hugokernel** [comment]
> @pacahon @hugokernel I've submitted a change that fixed this in my test template, and should not necessitate upgrading to the `translate` tag as above. Please test it out with your templates, as they might break differently than mine did.

I had try the new regexp and it works well.

Thx

**Comment by pacahon** [comment]
It works for me too. Thanks.
